<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationsWindow</class>
 <widget class="QDialog" name="NotificationsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notifcations</string>
  </property>
  <widget class="QTableWidget" name="NotificationTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteCurrentButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Selected Notification</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteAllButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete All Notifications</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseNotificationsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
